RELEASING.md

```diff
@@ -93,11 +133,15 @@ scm.tag=1.4.0
 
 ## Removing additional files
 
-Note: There is one additional step to remove the activemq-pom-<version>-source-release.zip from the Nexus staging repository before closing the staging repository.  At the moment this artifact is uploaded automatically by the Apache release plugin.  In future versions the ActiveMQ Artemis pom will be updated to take this into account.
+The last step before closing the staging repository is removing the activemq-pom-&lt;version>-source-release.zip.  At 
+the moment this artifact is uploaded automatically by the Apache release plugin. In future versions the ActiveMQ Artemis
```

Comment:
This can be avoided by overriding the profile and preventing the generation, e.g. see: https://github.com/apache/qpid-jms/blob/0.27.0/pom.xml#L326